assets/js/calypsoify.js

```diff
@@ -0,0 +1,44 @@
+( function( $ ) {
+    'use strict';
+
+    /**
```

Comment:
I suppose we could just wrap these in a `if (window.jpTracksAJAX)` but i'm likely being too paranoid :)

Reply:
Seems like a safe idea 😄